src/activeList.js

```diff
@@ -0,0 +1,153 @@
+const enterErr = document.querySelector('.enter-err');
+const edit = document.querySelector('.edit');
+const enterBtn = document.querySelector('.btn');
+const removeAll = document.querySelector('.removeAll');
+
+let itemsArr = [];
+
+const removeItem = function (item) {
+  const removeIndex = itemsArr.indexOf(item);
+  itemsArr.splice(removeIndex, 1);
+};
+
+const updateItem = function (currentItemIndex, value) {
+  const newItem = itemsArr[currentItemIndex];
+  newItem.description = value;
+  itemsArr.splice(currentItemIndex, 1, newItem);
+  // eslint-disable-next-line no-use-before-define
+  saveLocalStorage(itemsArr);
+};
+
+// eslint-disable-next-line func-names
+const handleItem = function (itemData) {
+  const items = document.querySelectorAll('.show-list');
+  items.forEach((item) => {
+    // eslint-disable-next-line eqeqeq
+    if (item.querySelector('.tast').getAttribute('data-time') == itemData.index) {
+      item.querySelector('[data-done]').addEventListener('change', (e) => {
+        e.preventDefault();
+        const itemIndex = itemsArr.indexOf(itemData);
+        const currentItem = itemsArr[itemIndex];
+        // eslint-disable-next-line no-unneeded-ternary
+        currentItem.completed = currentItem.completed ? false : true;
+        itemsArr.splice(itemIndex, 1, currentItem);
+        // eslint-disable-next-line eqeqeq
+        if (currentItem.completed == false) {
+          item.querySelector('.tast').style.textDecoration = '';
+          item.querySelector('.drag-drop').style.display = 'flex';
+          item.querySelector('.trash').style.display = 'none';
+        } else {
+          item.querySelector('.tast').style.textDecoration = 'line-through';
+          item.querySelector('.drag-drop').style.display = 'none';
+          item.querySelector('.trash').style.display = 'flex';
+        }
+        // eslint-disable-next-line consistent-return
+        removeAll.addEventListener('click', (e) => {
+          e.preventDefault();
+          // eslint-disable-next-line eqeqeq
+          if (currentItem.completed == true) {
+            const removeIndex = itemsArr.indexOf(currentItem);
+            itemsArr.splice(removeIndex, 1);
+          }
+          saveLocalStorage(itemsArr);
+        });
+        saveLocalStorage(itemsArr);
+      });
+      item.querySelector('[data-edit]').addEventListener('click', (e) => {
+        e.preventDefault();
+        inputF.value = itemData.description;
+        edit.innerHTML = 'Please Edit Your today\'s To-Do Task here';
+        document.querySelector('#objIndex').value = itemsArr.indexOf(itemData);
+      });
+      item.querySelector('[data-delete]').addEventListener('click', (e) => {
+        e.preventDefault();
+        parentNode.removeChild(item);
+        removeItem(item);
+        saveLocalStorage(itemsArr);
+        // eslint-disable-next-line eqeqeq
+        return itemsArr.filter((item) => item != itemData.description);
```

Comment:
I think it's better to use `!==` here

```suggestion
        return itemsArr.filter((item) => item !== itemData.description);
```